addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -8,7 +8,7 @@
         <div id="wrap">
             <t t-set="editor_message">DROP BUILDING BLOCKS HERE TO MAKE THEM AVAILABLE ACROSS ALL RESELLERS</t>
             <div class="oe_structure oe_empty" id="oe_structure_website_crm_partner_assign_layout_1" t-att-data-editor-message="editor_message"/>
-            <div class="container my-4">
+            <div class="container my-4 mw-75">
```

Comment:
In the spec it states "need a max-width **on the margin of the div** class "div.container.my-4" " , so not on the div itself. I think the idea was to give more room to the div ( here we are reducing the width for smaller screens actually :p )

But as LNA said, it should match other pages. The responsive width is actually linked to the 'container' selector, if I am not mistaken. So actually, I would remove the change and do nothing regarding that spec point :)

Reply:
Gotcha! I will make the changes quickly. Thanks :smile: